src/main/kotlin/dev/arbjerg/ukulele/jda/JdaConfig.kt

```diff
@@ -27,6 +27,7 @@ class JdaConfig {
 
         val intents = listOf(
                 GUILD_VOICE_STATES,
+                GUILD_PRESENCES,
```

Comment:
Avoid cache disable warning due to missing intent.  This and also messages require bot permissions in Developer Dashboard.